### ISSUE TITLE
[enhancement] add cli arguments

### ISSUE THREAD

**Issue by mb6ockatf** [IvanIsak2000/stalin_quotes#6]

**Comment by mb6ockatf** [comment]
@IvanIsak2000 could you please assign me to this issue, and add enhancement label? i have around 50 active issues, and it will help me manage them, really

same about #5

**Comment by IvanIsak2000** [comment]
I correct mean you need `argparser`?

**Comment by mb6ockatf** [comment]
@IvanIsak2000 yep, so you can choose all the things with arguments. at least, `--help` flag that will output some description of what is happening
upd. [in your program, i mean]
